core/src/main/java/com/linecorp/armeria/server/HttpServerHandler.java

```diff
@@ -233,13 +233,10 @@ private void cleanup() {
         if (!unfinishedRequests.isEmpty()) {
             final ClosedSessionException cause = ClosedSessionException.get();
             unfinishedRequests.forEach((req, res) -> {
+                // An HTTP2 request is cancelled by Http2RequestDecoder.onRstStreamRead()
+                final boolean cancel = !protocol.isMultiplex();
                 // Mark the request stream as closed due to disconnection.
-                if (!protocol.isMultiplex()) {
-                    // An HTTP2 request is cancelled by Http2RequestDecoder.onRstStreamRead()
-                    req.ctx().cancel(cause);
-                }
-                req.close(cause);
```

Comment:
Is it OK to remove this line?

Reply:
`req.abortResponse()` closes itself using `tryClose()`
https://github.com/line/armeria/blob/b9c203beff2958e8171eabb07f87b2631dace9e9/core/src/main/java/com/linecorp/armeria/server/DecodedHttpRequest.java#L162-L163